keyboards/handwired/ziyoulang_k3_mod/info.json

```
@@ -0,0 +1,146 @@
+{
+    "keyboard_name": "ziyoulang k3 mod",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xE002",
+        "vid": "0x5006"
+    },
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": false,
+        "audio": false,
+        "combo": false,
+        "midi": false,
+        "virtser": false,
+        "raw": false
```

It's generally preferred to have these features enabled, unless there's a good reason for them not to be.
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": false
```

---

BluePill has an edition of the STM32F103C6T6 which only has 22KB of available space, excluding the bootloader.
In order to reduce the size of Firmware, disabling mousekey and via LTO were necessary as minimum adjustments.

---

@waffle87 
Fixed. Please leave a review.